pkg/workflow_execution.go

```diff
@@ -437,32 +449,29 @@ func (c *Client) insertPreWorkflowExecutionStatistic(namespace, name string, wor
 		return 0, err
 	}
 
-	insertMap := sq.Eq{
-		"uid":                          uid,
-		"workflow_template_version_id": workflowTemplateVersionId,
-		"name":                         name,
-		"namespace":                    namespace,
-		"created_at":                   createdAt.UTC(),
-		"phase":                        wfv1.NodePending,
-		"parameters":                   string(parametersJSON),
-		"is_archived":                  false,
-	}
-
 	err = sb.Insert("workflow_executions").
-		SetMap(insertMap).
+		SetMap(sq.Eq{
+			"UID":                          UID,
```

Comment:
Lower case `uid` is ok as long as it's consistent across the system. If you're making this uppercase here, then you'd have to it [here](https://github.com/onepanelio/core/pull/374/files#diff-2df54c8de22b7ec30c64350c5cbfd591R536) and everywhere else too.